### PR TITLE
chore(ci): use the same node and npm version as we do in EVG ci

### PR DESCRIPTION
**Pull request by himanshusinghs**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: .github/workflows/authors-and-third-party-notices.yaml [hunk 1]
```diff
@@ -23,14 +23,14 @@ jobs:
           fetch-depth: '0'
 
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: ^16.20.2
+          node-version: ^18.19.1
```
Comment (gribnoysup) [inline .github/workflows/authors-and-third-party-notices.yaml:28]: We use exact, not range

Comment (himanshusinghs) [reply .github/workflows/authors-and-third-party-notices.yaml:28]: Ahh wow, I didn't check that. Thanks - I will check and fix.

Comment (lerouxb) [reply .github/workflows/authors-and-third-party-notices.yaml:28]: wasn't it `^16.20.2` before which is a range?

Comment (gribnoysup) [reply .github/workflows/authors-and-third-party-notices.yaml]: It's not a range in evergreen

File: .github/workflows/bump-packages.yaml
[the remark this answers is not in the document]
Comment (himanshusinghs) [reply .github/workflows/bump-packages.yaml:1]: Noice! Thanks for the pointer, I just pushed an update. Didn't look deeper if the params have changed but if I see the action failing. I will fix it.

[the remark this answers is not in the document]
Comment (gribnoysup) [reply .github/workflows/bump-packages.yaml:1]: Thanks!

**Review by gribnoysup — APPROVED** [review]

**Comment by himanshusinghs** [comment]
Merging now since all the GH checks went fine. The EVG checks are not related.

File: .github/workflows/authors-and-third-party-notices.yaml (hunk 1 above)
Comment (lerouxb) [reply .github/workflows/authors-and-third-party-notices.yaml]: Oh you mean in evergreen. Forgot this is to make it the same as there and not just to bump it. Carry on, then ;)

**Pull request merged**